### PR TITLE
Added guest blog page

### PR DESCRIPTION
**Pull request by farfamahar**
-Added guestblog page
-Removed uppercase styling from blog titles
-moved some inline css to style.css
-minor padding, margin, and font size changes

### CODE REVIEW

**Comment by farfamahar** [comment]
I'm not fully sure how page templates and routing works still 😕 but I changed my local development routes file to [this](https://pastebin.com/DjdJcBr0) and added a guestblog page. My thinking right now is to have a small paragraph at the top of the blog page which contains a link to the guestblog page.

**Comment by backspace** [comment]
what do you think of using tags to mark guest posts and having them mixed in with the rest of them? I don’t know if we got much detail from people about what they want to see here 😞

re the routes file, if you replace `.content-overrides/routes.yaml` with your changes it’ll be installed in the preview deployment and also lets us make sure we have the routes stored in source control vs the weird unversioned way they’re stored on the production server 🤓

**Comment by farfamahar** [comment]
> what do you think of using tags to mark guest posts and having them mixed in with the rest of them? I don’t know if we got much detail from people about what they want to see here 😞
> 

Yeah, I think making use of tags would be a good idea. I should clarify that the guesblog page I made is just a page that just has information on how to submit a guestblog. It's basically the same as the page template but uses a smaller font. You know, now that I think about it, we could probably just make a blog post about that information, and link to the blog post in a paragraph at the top of the blog page.

Edit - The blog page before used the index file and would only display posts and not use the page-blog file. Now it uses the page-blog file and can have user editable text at the top like donate, home and resource page. Although for some reason I can't get pagination to work on it like it was in the index file.

File: page-resources.hbs [hunk 1]
```diff
@@ -41,14 +41,14 @@
         
     <div id="mission" data-aos="fade-up" class="section-1">
 
-        <div class="text-heading" > <h2 style="margin-top: 100px;">Posters</h2></div>
+        <div class="text-heading text-upper" > <h2> RESOURCES </h2></div>
```
Comment (backspace) [inline page-resources.hbs]: It’s redundant to have `.text-upper` and a capitalised string inside the `h2`, right? And above on `page-contact` 🤓

File: page-resources.hbs [hunk 2]
```diff
@@ -41,14 +41,14 @@
...
-        <div class="text-heading" > <h2 style="margin-top: 100px;">Posters</h2></div>
+        <div class="text-heading text-upper" > <h2> RESOURCES </h2></div>
         <div class="text-body"> {{#page}} {{content}} {{/page}} </div>
         
     </div>
 
     <section class="flex-grid-thirds" style="margin-bottom: 50px">
 
-        <a href="   https://drive.google.com/file/d/1DCkjFweTrxfWZcsn8bDnJhh1nAev0Wmh/view" target="_blank"><div data-aos="fade-up" class="left "> <img class="responsive-poster-size hvr-float"  src="{{asset "images/poster1.png"}}" width="410" height="532">
+        <a href="   https://drive.google.com/file/d/1DCkjFweTrxfWZcsn8bDnJhh1nAev0Wmh/view" target="_blank"><div data-aos="fade-up" class="left "> <img class="responsive-poster-size hvr-float"  src="{{asset "images/poster1.png"}}"  width="410" height="532">
```
Comment (backspace) [inline page-resources.hbs:51]: quite minor but it’s sometimes nice to excise “noise” like this, I assume it’s not a deliberate change?

File: assets/css/style.css [hunk 3]
```diff
@@ -916,20 +986,35 @@ color:white !important;
...
     text-align: center;
     font-family: 'Open Sans', sans-serif;
+    word-wrap: break-word;
+
+}
+
+.text-body-20px {
+    padding: 100px;
+    font-size:20px;
+    font-family: 'Open Sans', sans-serif;
```
Comment (backspace) [inline assets/css/style.css]: It seems like this isn’t used anywhere? 🤔

File: assets/css/style.css [hunk 4]
```diff
@@ -510,6 +509,47 @@ html[data-current="contact"] main > form button {
...
+  color: #555555;
+  padding:1em;
+  border-left:8px solid #DE3545 ;
+  line-height:1.6;
+  position: relative;
+  background:#EDEDED;
+}
+
+blockquote::before{
+  font-family:Arial;
```
Comment (backspace) [inline assets/css/style.css]: What does setting the font here do?

I like the new look of `blockquote`, it’s great on a blog post:

image

But maybe needs tweaking on a page?

image

**Review by backspace — APPROVED** [review]
Sorry for taking so long on this. I sometimes struggle with reviewing changes when I have to run them locally, especially when they’re CSS-heavy. I’ll look into whether I can bring in [Percy](https://percy.io/) for visual diffs to make it easier to review.

I understand now re the intent of the guest blog page. My general desire in theme work is to empower people to accomplish as much as possible through the admin interface, but sometimes custom templates are still necessary.

I think there are a few places this could be improved but also I think it would be fine to go ahead with this as is and make refinements in future PRs. The blog pagination is something we should return to but I think it’s fine to have it not quite working for now.

File: page-resources.hbs (hunk 2 above)
Comment (farfamahar) [reply page-resources.hbs:51]: Oops. I was going to add alt text, but decided I'd add it in another pull request.

File: assets/css/style.css (hunk 3 above)
Comment (farfamahar) [reply assets/css/style.css]: Yeah, this isn't used. Removing

File: assets/css/style.css (hunk 4 above)
Comment (farfamahar) [reply assets/css/style.css]: Looks like the font setting does nothing. Removed. Will need to tweak block quotes on pages in a future pr. Thanks.

File: page-resources.hbs (hunk 1 above)
Comment (farfamahar) [reply page-resources.hbs]: Yeah, removed now. Thanks.

**Review by farfamahar — COMMENTED** [review]
Thanks for reviewing this! 👍  I'm going to commit these changes and merge them.

**Pull request merged**